tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -20,7 +20,7 @@ commands:
       RETURN_CODE=0
       PUBLIC_URL=$(oc get -n $NAMESPACE horizon horizon -o jsonpath='{.status.endpoint}')
```

Comment:
So this one successfully gets the endpoint from the statue:
```
logger.go:42: 11:15:54 | basic-deployment/1-deploy-horizon | ++ oc get -n horizon-kuttl-tests horizon horizon -o 'jsonpath={.status.endpoint}'
    logger.go:42: 11:15:54 | basic-deployment/1-deploy-horizon | + PUBLIC_URL=http://horizon.horizon-kuttl-tests.svc/

```

This one fails:
```
    logger.go:42: 11:15:54 | basic-deployment/1-deploy-horizon | ++ oc exec -it deployment/horizon -- curl --silent --output /dev/stderr --head --write-out '%{http_code}' 'http://horizon.horizon-kuttl-tests.svc/dashboard/auth/login/?next=/dashboard/' -k
    logger.go:42: 11:15:54 | basic-deployment/1-deploy-horizon | Error from server (NotFound): namespaces "ci-op-tt7vckqd" not found
```

It just needs the `-n $NAMESPACE` in the command.

Reply:
Sorry, no change required for this line. Just highlighting that this command works and it contains the `-n $NAMESPACE` while the next command doesn't and is missing the `-n $NAMESPACE`